tests/test_jobcollection.py

```diff
@@ -18,10 +22,86 @@ def test_jobcollection(jobcollection_mock):
     assert len(jobcollection_mock.jobs) == 1
 
 
+def test_jobcollection_multiple(jobcollection_multiple_mock):
+    assert len(jobcollection_multiple_mock.jobs) == 2
+
+
+def test_jobcollection_download_result(jobcollection_mock):
+    with requests_mock.Mocker() as m:
+        download_url = "http://up42.api.com/abcdef"
+        url_download_result = (
+            f"{jobcollection_mock.auth._endpoint()}/projects/"
+            f"{jobcollection_mock.project_id}/jobs/{jobcollection_mock.jobs_id[0]}/downloads/results/"
+        )
+        m.get(url_download_result, json={"data": {"url": download_url}, "error": {}})
+
+        out_tgz = Path(__file__).resolve().parent / "mock_data/result_tif.tgz"
+        out_tgz_file = open(out_tgz, "rb")
+        m.get(
+            url=download_url,
+            content=out_tgz_file.read(),
+            headers={"x-goog-stored-content-length": "163"},
+        )
+
+        with tempfile.TemporaryDirectory() as tempdir:
+            out_dict = jobcollection_mock.download_results(tempdir, merge=False)
+            for job_id in out_dict:
+                assert Path(out_dict[job_id][0]).exists()
+                assert len(out_dict[job_id]) == 2
+
+
+def test_jobcollection_download_result_merged(jobcollection_multiple_mock):
+    with requests_mock.Mocker() as m:
+        download_url = "http://up42.api.com/abcdef"
+        url_download_result_1 = (
+            f"{jobcollection_multiple_mock.auth._endpoint()}/projects/"
+            f"{jobcollection_multiple_mock.project_id}/jobs/{jobcollection_multiple_mock.jobs_id[0]}/downloads/results/"
+        )
+        url_download_result_2 = (
+            f"{jobcollection_multiple_mock.auth._endpoint()}/projects/"
+            f"{jobcollection_multiple_mock.project_id}/jobs/{jobcollection_multiple_mock.jobs_id[1]}/downloads/results/"
+        )
+        m.get(url_download_result_1, json={"data": {"url": download_url}, "error": {}})
+        m.get(url_download_result_2, json={"data": {"url": download_url}, "error": {}})
+
+        out_tgz = Path(__file__).resolve().parent / "mock_data/result_tif.tgz"
+        out_tgz_file = open(out_tgz, "rb")
+        m.get(
+            url=download_url,
+            content=out_tgz_file.read(),
+            headers={"x-goog-stored-content-length": "163"},
+        )
+
+        with tempfile.TemporaryDirectory() as tempdir:
```

Comment:
```suggestion
        with tempfile.TemporaryDirectory() as tmpdir:
```
tempfile package contains `tempdir` method, just to prevent any conflict with in-built function naming!

Reply:
Right, didn't think of this, changed!